### ISSUE TITLE
[Bug Report] Mujoco environment : Render Window Crashes upon pressing a key

### ISSUE THREAD

**Issue by Arckonte** [openai/gym#2860]
When rendering a mujoco environment I can move the camera using the mouse, but the render window crashes when I press any key.
I also have to manualy call glfw.terminate() to close the window when I'm done using the environment, weither it has crashed or not.
Is it a bug, or am I supposed to pass arguments on the env.render() methods ? I saw no mention of it on the documentation.

```
\testMujoco.py:28 in <module>
    env.render()
  File D:\Programmes\Miniconda3\lib\site-packages\gym\core.py:328 in render
    return self.env.render(**kwargs)
  File D:\Programmes\Miniconda3\lib\site-packages\gym\wrappers\order_enforcing.py:51 in render
    return self.env.render(**kwargs)
  File D:\Programmes\Miniconda3\lib\site-packages\gym\envs\mujoco\mujoco_env.py:233 in render
    self._get_viewer(mode).render()
  File D:\Programmes\Miniconda3\lib\site-packages\gym\envs\mujoco\mujoco_rendering.py:546 in render
    update()
  File D:\Programmes\Miniconda3\lib\site-packages\gym\envs\mujoco\mujoco_rendering.py:525 in update
    glfw.poll_events()
  File D:\Programmes\Miniconda3\lib\site-packages\glfw\__init__.py:1844 in poll_events
    _glfw.glfwPollEvents()
  File D:\Programmes\Miniconda3\lib\site-packages\glfw\__init__.py:681 in errcheck
    _reraise(exc[1], exc[2])
  File D:\Programmes\Miniconda3\lib\site-packages\glfw\__init__.py:70 in _reraise
    raise exception.with_traceback(traceback)
  File D:\Programmes\Miniconda3\lib\site-packages\glfw\__init__.py:660 in callback_wrapper
    return func(*args, **kwargs)
TypeError: _key_callback() takes 3 positional arguments but 6 were given
```

Here his a simple code that can be used to reproduce the bug: 

```
import gym
import glfw

# env = gym.make("Ant-v4")
# env = gym.make("Humanoid-v4")
# env = gym.make("Reacher-v4")
# env = gym.make("InvertedDoublePendulum-v4")
# env = gym.make("InvertedPendulum-v4")
# env = gym.make("Hopper-v4")
# env = gym.make("HumanoidStandup-v4")
# env = gym.make("Walker2d-v4")
# env = gym.make("Swimmer-v4")
env = gym.make("Ant-v4")

env.reset()
for i in range(1000):
  #renders the environment
  env.render()
  action = env.action_space.sample()
  (o, r, done, info) = env.step(action)
  if done:
      env.reset()

glfw.terminate()
env.close()
```

I am using Gym 0.24.0 (so with the new Mujoco bindings) on anaconda3 on windows10

**Comment by rodrigodelazcano** [comment]
Hi @Arckonte thanks for bringing this up. I made this PR  https://github.com/openai/gym/pull/2861 that fixes the key error. I wasn't able to replicate the closing window issue but might have fixed it by calling the glfw.terminate() when the environment is done. 
Could you please try the code on this PR on your end?
These fixes will be included in the next Gym release v24.1 or they can be install directly from master once this PR is merged.

**Comment by Arckonte** [comment]
Thanks for your help @rodrigodelazcano ! This indeed fixes the issue.
I didn't find out how to install the PR#2861, so I waited for the 24.1 release instead.
